string order_id
string phone
string address
---
string qr_base64
